packages/garbo/src/tasks/post/index.ts

```diff
@@ -339,11 +342,27 @@ function handleDrenchedInLava(): GarboPostTask {
   };
 }
 
+function acquireAbortFreeRun(): GarboPostTask {
```

Comment:
I don't love the way this function is designed. I think it would make more sense to have the `bestFreeRun` be like a cached value returned by a function. I guess it overall makes sense to have it in post, but something about the calculation logic here being done in the task-making function feels wrong to me for reasons I can't articulate.